config/local.template.yml

```diff
@@ -0,0 +1,8 @@
+# Copy this file, rename it to `local.yml`, and enter your credentials to run the bot.
+arisa:
+  credentials:
+    username: <JIRA user name>
+    password: <plain text password>
+    dandelionToken: <Dandelion token>
```

Comment:
This is no longer mandatory, so we can put a comment with optional

Reply:
Added a comment there, and added the other optional credentials too